### ISSUE TITLE
Typo makes parser error handler trip

### ISSUE THREAD

**Issue by bartv** [inmanta/inmanta#640]
```
typedef positive as number matching self >= 1-
```

The last - gives a syntax error without any location.

```
Syntax error -
 ()
```

**Comment by sanderr** [comment]
I believe this is solved by #1706 (fixing #1341). The message now reads:
```
Syntax error: Illegal character '-' (./main.cf:1:46)
```

**Comment by sanderr** [comment]
#1706 is merged in. Can this issue be closed?

**Comment by wouterdb** [comment]
fixed in #1706
